backend/src/test/java/com/woowacourse/friendogly/footprint/domain/FootPrintTest.java

```diff
@@ -61,13 +64,14 @@ void findChangedWalkStatusTime_when_WalkStatusIsOngoing(){
     void findChangedWalkStatusTime_when_WalkStatusIsAfter(){
         // given
         LocalDateTime expectedTime = LocalDateTime.of(2023, 12, 12, 10, 30);
```

Comment:
```suggestion
        LocalDateTime createdAt = LocalDateTime.of(2023, 12, 12, 10, 30);
```

직관적인 이름이 더 좋지 않을까 싶어요
`이 테스트의 기댓값`이라는 네이밍도 좋지만, "그래서 이 테스트에서는 어떤 값을 기대하는데?" 라는 생각이 한 번 더 들 수도 있을 것 같아요!

Reply:
이 테스트에서 기대되는 반환 시간은 createdAt이 아닙니다! 
`산책후일경우, 산책 상태 변경된 시간은 산책 종료 시간이 반환된다.`
여기서 말하는 expectedTime 은 산책종료시간에 들어갈 time입니다.
일단 트레가 혼동한 것부터 네이밍이 명확하지 않다는 것은 파악할 수 있었네요. 조금 더 고민해볼게요

Reply:
`Footprint`는 빌더 패턴 괜찮아 보이네요!!